.github/workflows/cppinterop-diff.yml

```
@@ -0,0 +1,26 @@
+name: Diff CppInterOp against upstream
+on:
+  pull_request:
+    paths:
+      - 'interpreter/CppInterOp/**'
+
+jobs:
+  cppinterop-diff:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out ROOT
+        uses: actions/checkout@v4
+        with:
+          path: root
+      - name: Check out CppInterOp
+        uses: actions/checkout@v4
+        with:
+          repository: compiler-research/CppInterOp
+          path: CppInterOp
+      - name: Drop directories that are not added to ROOT
+        working-directory: CppInterOp
+        run: |
+          rm -rf .github
+      - name: Compare
+        continue-on-error: true
```

Ah, I was already wondering: With this line you just ignore the error and continue even if the two copies have a difference. That renders the check useless...

---

Thanks, I will drop the continue-on-error setting. I will note that we may have to keep a patch disabling a unittest, assuming the other failing test on Windows is fixed and no longer disabled